client/src/test/java/io/confluent/kafka/schemaregistry/client/rest/RestServiceTest.java

```diff
@@ -31,6 +31,7 @@
 import java.net.URL;
 import java.nio.charset.StandardCharsets;
 
+import avro.shaded.com.google.common.collect.ImmutableMap;
```

Comment:
It seems brittle to rely on a shaded class, especially one in Avro (since later these tests should be format-independent)